geoh5py/objects/curve.py

```diff
@@ -174,7 +174,7 @@ def remove_cells(self, indices: list[int]):
         """Safely remove cells and corresponding data entries."""
 
         if self._cells is None:
-            warnings.warn("No cells to be removed.")
+            warnings.warn("No cells to be removed.", UserWarning)
```

Comment:
No need - warnings.warn already uses an instance of UserWarning.

![image](https://user-images.githubusercontent.com/81254271/193860603-1d0c7d1b-e82a-4711-9bad-dfb2a83a978b.png)

Reply:
I thought so, but I got an error on gitactions without it

Reply:
https://github.com/MiraGeoscience/geoh5py/actions/runs/3177338732/jobs/5177662741#step:9:79